src/server/game/Spells/Auras/SpellAuras.cpp

```diff
@@ -744,6 +744,72 @@ void Aura::UpdateOwner(uint32 diff, WorldObject* owner)
     _DeleteRemovedApplications();
 }
 
+void Aura::Update(uint32 diff, Unit* caster)
```

Comment:
this function allready exists on line 813

Reply:
instead of tabs use 4 spaces pls setup your configs in msvc

Reply:
most of the scripts need to be rewriten and linked with db most of them never worked